Documentation/development-environment.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Rook development workflow on complex multi-node env
+weight: 51
+---
+
+# Rook development workflow
+
+## Setup expectation
+
+There are a bunch of pre-requisites to be able to deploy the following environment. Such as:
+
+* A Linux workstation (CentOS or Fedora)
+* KVM/QEMU installation
+
+For other Linux distribution, there is no guarantee the following will work.
+However adapting commands (apt/yum/dnf) could just work.
+
+## Prerequisites installation
+
+Install the following packages on the machine that will run the Kubernetes env:
+
+```bash
+sudo yum install -y qemu libvirt libvirt-devel ruby-devel gcc qemu-kvm docker
```

Comment:
what if we put this in a script in `tests/scripts` next to minikube.sh? then the documentation could just refer to the script to bring up this type of multi-node environment. Or maybe we create a subfolder such as `/tests/scripts/multinode` so we can create the necessary scripts, Vagrantfile, etc. Much of this doc looks like it could go in there.

Reply:
Agree, this is more a brain dump for now, but yes I'm planning on doing some automation too :).